docs/reference/edot-sdks/android/configuration.md

```diff
@@ -0,0 +1,380 @@
+description: Configure the Elastic Distribution of OpenTelemetry Android (EDOT Android) to send data to Elastic.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-sdk
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/android/current/configuration.html
+---
+
+# Configure the EDOT Android SDK [configuration]
+
+The following sections contain a comprehensive list of all the configurable parameters available for the SDK, including those you can set during initialization and those you can adjust dynamically afterward.
+
+## Initialization configuration 
+
+Initialization configuration is available from the SDK builder shown in [Agent setup](getting-started.md#agent-setup). The following are its available parameters.
+
+### Application metadata
+
+Provide your application name, version, and environment. For example:
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            .setServiceName("My app name") // <1>
+            .setServiceVersion("1.0.0") // <2>
+            .setDeploymentEnvironment("prod") // <3>
+            // ...
+            .build()
+    }
+}
+```
+
+1. Name used by {{kib}} when listing your application on the [Services](docs-content://solutions/observability/apm/services.md) page. Defaults to `unknown`. Refer to [Why your app is referred to as a "service"](troubleshooting.md#why-service).
+2. Your app's version name. Defaults to the version provided [in the PackageInfo](https://developer.android.com/reference/android/content/pm/PackageInfo#versionName).
+3. Your app's build type, flavor, backend environment, or a combination of these. Helpful to better analyze your app's data later in {{kib}}.
+
+### Export connectivity
+
+Configure where your app's telemetry is exported:
+
+```kotlin
+class MyApp : android.app.Application {
+
+    override fun onCreate() {
+        super.onCreate()
+        val agent = ElasticApmAgent.builder(this)
+            // ...
+            .setExportUrl("https://my-elastic-apm-collector.endpoint") // <1>
+            .setExportAuthentication(Authentication.ApiKey("my-api-key")) // <2>
+            .setExportProtocol(ExportProtocol.HTTP) // <3>
+            .build()
+    }
+}
+```
+
+1. Your endpoint URL. If you don't have one yet, refer to [Get the export endpoint](getting-started.md#get-export-endpoint).
+2. Your authentication method. You can use either an [API key](docs-content://solutions/observability/apm/api-keys.md), a [Secret token](docs-content://solutions/observability/apm/secret-token.md), or none. Defaults to `None`. API keys are the recommended method, if you don't have one yet, refer to [Create an API key](getting-started.md#create-api-key).
+3. The protocol used to communicate with your endpoint. It can be either `HTTP` or `gRPC`. Defaults to `HTTP`.
+
+:::{tip}
```

Comment:
nit: Can we use [file inclusion](https://elastic.github.io/docs-builder/syntax/file_inclusion/) for this tip? Since it's also present in getting-started, I think it's better to avoid duplicating it.